### ISSUE TITLE
malt - make install fails

### ISSUE THREAD

**Issue by ednospam** [themetaschemer/malt#5]
Unable to install malt using instructions from thelittlelearner.com from git using make/install. Notice make and tests pass.

$ cd learner/
$ git clone https://github.com/themetaschemer/malt.git 
Cloning into 'malt'...
remote: Enumerating objects: 533, done.
remote: Counting objects: 100% (533/533), done.
remote: Compressing objects: 100% (291/291), done.
remote: Total 533 (delta 300), reused 458 (delta 227), pack-reused 0
Receiving objects: 100% (533/533), 1.28 MiB | 7.87 MiB/s, done.
Resolving deltas: 100% (300/300), done.
$ cd malt

 $ make
Building ...
Tensor implementation: flat-tensors
Tensor implementation (no-duals): flat-tensors
Tensor implementation (no-overrides): flat-tensors
Tensor implementation (no-duals, no-overrides): flat-tensors
Running tests ...
571 tests passed

$ make install
"/usr/bin/raco" pkg install
Linking current directory as a package
build-path: contract violation
  expected: (or/c path-for-some-system? path-string? 'up 'same)
  given: #rx"test\\\\"
  argument position: 2nd
  other arguments...:
   #<path:/home/pi/learner/malt/>
  context...:
   /usr/share/racket/collects/pkg/private/addl-installs.rkt:28:2: loop
   /usr/share/racket/collects/pkg/private/stage.rkt:107:0: stage-package/info
   /usr/share/racket/collects/pkg/private/install.rkt:141:0: install-packages
   /usr/share/racket/collects/pkg/private/install.rkt:925:4
   /usr/share/racket/collects/racket/private/more-scheme.rkt:206:2: call-handled-body
   /usr/share/racket/collects/pkg/private/prefetch.rkt:54:0: call-with-prefetch-cleanup
   /usr/share/racket/collects/racket/contract/private/arrow-val-first.rkt:555:3
   /usr/share/racket/collects/racket/file.rkt:437:8
   /usr/share/racket/collects/racket/private/kw-file.rkt:116:2: call-with-output-file*
   /usr/share/racket/collects/racket/file.rkt:426:0: call-with-file-lock
   /usr/share/racket/collects/pkg/main.rkt:216:16
   /usr/share/racket/collects/racket/private/more-scheme.rkt:206:2: call-handled-body
make: *** [Makefile:262: install] Error 1

Environment
Linux raspberrypi 5.15.84-v8+ #1613 SMP PREEMPT Thu Jan 5 12:03:08 GMT 2023 aarch64 GNU/Linux
Racket v7.9 [bc].

**Comment by themetaschemer** [comment]
Thanks for reporting the issue @ednospam. Looks like it is a Racket versioning issue. I just found that one of the features we're using from Racket to support cross platform portability was only introduced in a later version: 
`Changed in version 8.1.0.5: Added support for regexps in compile-omit-paths.`

Would it be possible for you to upgrade you Racket to 8.2 or higher. I will update the documentation and code as well to reflect this requirement.
